com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl

```diff
@@ -382,18 +406,11 @@ half MainLightShadow(float4 shadowCoord, float3 positionWS, half4 shadowMask, ha
 #endif
 
 #ifdef MAIN_LIGHT_CALCULATE_SHADOWS
-    half shadowFade = GetShadowFade(positionWS);
+    half shadowFade = GetShadowFade(positionWS, shadowCoord.w);
 #else
     half shadowFade = 1.0h;
 #endif
 
-#if defined(_MAIN_LIGHT_SHADOWS_CASCADE) && defined(CALCULATE_BAKED_SHADOWS)
-    // shadowCoord.w represents shadow cascade index
-    // in case we are out of shadow cascade we need to set shadow fade to 1.0 for correct blending
-    // it is needed when realtime shadows gets cut to early during fade and causes disconnect between baked shadow
-    shadowFade = shadowCoord.w == 4 ? 1.0h : shadowFade;
-#endif
-
```

Comment:
I tested, it is no longer needed as we are calculating shadow fade from cascade

This was the problem.
![image](https://user-images.githubusercontent.com/24258017/104744998-3d2b8b80-5756-11eb-8af8-0d3dbc32e453.png)